packages/runtime/runtime-utils/src/utils.ts

```diff
@@ -27,3 +28,43 @@ export async function seqFromTree(
 	const attrib = await readAndParseBlob<IDocumentAttributes>(attributesHash);
 	return attrib.sequenceNumber;
 }
+
+/**
+ * Encode compact ID (returned by IContainerRuntime.generateDocumentUniqueId()) to a compact string representation.
+ * While this is the main usage pattern, it works with any non-negative integer or a string.
+ * Strings are retured as is, and assumed to be UUIDs, i.e. unique enough to never overlap with
+ * numbers encoded as strings by this function. Any other strings are likely to run into collisions and should not be used!
+ * This function is useful in places where we serialize resulting ID as string and use them as strings, thus we are not
+ * gaining any efficiency from having a number type.
+ * We do not provide a decode function, so this API is only useful only result is stored and there is no need to go back to origianl form.
+ * @param numArg - input number, non-negative integer.
```

Comment:
Update this comment because this argument can be a string as well. Maybe "compactId"?

Reply:
Also, why not just make it numbers since passing string is a no-op?

Reply:
Changed it. This is mostly for easier usage - i.e. encodeCompactIdToString(generateDocumentUniqueId()).
If it works only with numbers, then consumers will need to have that code:

const id = generateDocumentUniqueId();
return typeof id === "string" ? id : encodeCompactIdToString(id)

It's fine, but I'd rather us to provide such function.
Maybe that's a better flow - provide such stand-alone function, i.e.


getCompactId(containerRuntime) -> string ?